src/widgets/survey/model/mappers.ts

```diff
@@ -432,6 +432,9 @@ function convertToAnswerDto(type: ActivityItemType, answer: Answer): AnswerDto {
     case 'TextInput':
       return convertToTextAnswer(answer);
 
+    case 'ParagraphText':
+      return convertToTextAnswer(answer);
```

Comment:
Please create a separate mapper for `ParagraphText` to ensure nothing will break in case of any changes.